syntax = "proto3";
package mesh;

message PingRequest {
  string message = 1;
}

message PingResponse {
  string message = 1;
}
